### ISSUE TITLE
Cloud Version

### ISSUE THREAD

**Issue by frunika** [OpenTrafficCam/OTVision#47]
Make the detection (and tracking) ready for AWS, Azure, Cloud computing, ...

It would be possible to scale the expensive stuff for larger projects and there is no need for dedicated hardware on site.

Big question: privacy!

**Comment by martinbaerwolff** [comment]
... and ready for high performance clusters @universities

**Comment by frunika** [comment]
Yep. It could be another task, depending on the management system.
